tools/build-image.sh

```diff
@@ -81,6 +90,21 @@ done
 [[ -z "${build_name}" ]] && \
   usage_error_exit "missing build name"
 
+[[ -z "${imgtype}" ]] && \
+  usage_error_exit "image type must be provided"
```

Comment:
nit: not sure this is really necessary since there is a default value for _imgtype_.